src/config/dynamic-theme-fixes.config

```diff
@@ -2664,6 +2664,17 @@ CSS
 
 ================================
 
+bulldogjob.pl
+
+INVERT
+img[alt="Bulldogjob"]
+img[alt="Bulldogjob for IT"]
+img[alt="Bulldogjob - The Space for IT People"]
+img[src="/bulldog.guide.svg"]
+img[src="/bulldog.tte.icon.svg"]
```

Comment:
Are all of these different CSS selector required? Couldn't it be one selector?

Reply:
Oh yes I can merge the selectors responsible for the logo into one. I will update the pull request soon.